README.md

```diff
@@ -162,6 +162,13 @@ server-tools: # Here we repeat server-tools, but no problem because it's a
     - html_text
 ```
 
+You can add all modules in a repo by using a `*`:
+
+```yaml
+website:
+    - "*"
```

Comment:
Ohhhh ok I remember why I didn't implement this - it will require a redesign of the way that the logic works in some of the loops. I admittedly didn't want to get into that yet because I don't need it yet.

This is also why I left it at the current syntax though, which will allow for an easier feature addition.

Are you opposed to just adding this to Roadmap for in incremental improvement & leaving the current syntax as is?